desc/integrals/quad_utils.py

```diff
@@ -298,7 +298,7 @@ def chebgauss2(deg):
     # github.com/scipy/scipy/blob/v1.14.1/scipy/special/_orthogonal.py#L1803-L1851.
     t = jnp.arange(deg, 0, -1) * jnp.pi / (deg + 1)
     x = jnp.cos(t)
-    w = jnp.pi * jnp.abs(jnp.sin(t)) / (deg + 1)
+    w = jnp.pi * jnp.sin(t) / (deg + 1)
```

Comment:
why the removal of the abs? shouldnt this be positive?

Reply:
the domain `t` makes this positive regardless. less operations means less gradients so less memory